ARGO/include/AssetManager.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <SDL.h>
+#include <map>
+#include <Utilities.h>
+
+class AssetManager
+{
+public:
+	static AssetManager* Instance(SDL_Renderer& t_renderer);
+	static AssetManager* Instance();
+	static void Release();
+
+	SDL_Texture* GetTexture(std::string t_filename);
+	Mix_Chunk* GetSfx(std::string t_filename);
+	Mix_Music* GetMusic(std::string t_filename);
+	SDL_Texture* GetText(std::string t_text, std::string t_filename, int t_size, SDL_Color t_colour);
+	TTF_Font* GetFont(std::string t_filename, int t_fontSize);
+
+	AssetManager(SDL_Renderer& t_renderer);
```

Comment:
why are the constructor/destructor below everything else?

Reply:
they're meant to be private actually, forgot to change it